assets/queries/ansible/gcp/sql_db_instance_with_ssl_disabled/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "SQL DB Instance With SSL Disabled",
   "severity": "HIGH",
   "category": "Encryption",
-  "descriptionText": "Cloud SQL Database Instance with SSL disabled for incoming connections",
+  "descriptionText": "Cloud SQL Database Instance should have SLL enabled",
```

Comment:
Can you also update it in `kics/assets/queries/terraform/gcp/sql_db_instance_with_ssl_disabled/metadata.json`, please?

Reply:
Done!